src/UuidBinaryOrderedTimeType.php

```diff
@@ -12,6 +12,7 @@
 
 namespace Ramsey\Uuid\Doctrine;
 
+use Doctrine\DBAL\ParameterType;
```

Comment:
Overall fine to have this patch, but then we need to bump the minimum supported `doctrine/dbal` to `^3`.